terraform/services/bucket-access-logs/main.tf

```
@@ -0,0 +1,72 @@
+resource "aws_s3_bucket" "bucket_access_logs" {
+  bucket        = "${var.app}-${var.env}-bucket-access-log"
```

Since we're creating just one bucket per account, let's use the account ID in the bucket name. Add an [aws_caller_identity data block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity) and reference the account ID like so:

```suggestion
  bucket        = "${data.aws_caller_identity.current.account_id}-bucket-access-logs"
```